### PR TITLE
Remove onSelectedCellChange prop

### PR DESCRIPTION
**Pull request by amanmahajan7**

### CODE REVIEW

File: src/hooks/useRovingRowRef.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { css } from '@linaria/core';
 import { cell } from '../style';
 import { useFocusRef } from './useFocusRef';
 
-const rowSelected = css`
+export const rowSelected = css`
```
Comment (nstepien) [inline src/hooks/useRovingRowRef.ts:6]: Just use `rdg-row-selected` in the tests
```suggestion
const rowSelected = css`
```

**Review by nstepien — APPROVED** [review]

**Comment by chilind** [comment]
I'm also using `onSelectedCellChange` and wondering if there is a replacement prop?

**Comment by chilind** [comment]
I'm also using `onSelectedCellChange` and wondering if there is a replacement prop?

**Comment by SergeyKatugin** [comment]
@nstepien we also used this props, mb have another prop to handle selected cell change?

**Pull request merged**